ui/src/theme/globals.ts

```diff
@@ -1,18 +1,29 @@
 import { globalCss } from '@stitches/react';
 
 export const themeGlobals = globalCss({
-  'html, body, #root': {
+  'html, body': {
     height: '100%',
-    padding: 0,
+
+    //TODO add theme colors
     color: '#ECEDEE',
     backgroundColor: 'black',
 
     fontFamily: 'Manrope',
-
     fontSize: '16px',
 
+    '* > *': {
+      boxSizing: 'border-box',
+    },
+

```

Comment:
we may use like that but outside the `'html, body'` key
```suggestion
    '*': {
      boxSizing: 'border-box',
    },
```